deploy/steth.ts

```diff
@@ -0,0 +1,139 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types";
```

Comment:
As you mentioned offline, looks like this is copy-pasta. I'll review this once it's updated.